autogen/main/variables.tf.tmpl

```diff
@@ -524,10 +524,16 @@ variable "disable_default_snat" {
   default     = false
 }
 
-variable "notification_config_topic" {
-  type = string
-  description = "The desired Pub/Sub topic to which notifications will be sent by GKE. Format is projects/{project}/topics/{topic}."
-  default = ""
+variable "notification_config" {
```

Comment:
nit: since this would be a breaking change due to var type change, could we add a new `notification_config_topic_types` top level var instead?